composer.lock

```diff
@@ -77,43 +77,37 @@
         },
         {
             "name": "laminas/laminas-eventmanager",
-            "version": "3.2.1",
+            "version": "3.4.0",
```

Comment:
I  guess, here is the problem.
you should not run `composer update` in these PR.
we make PR just for these kind of update.

Reply:
ok thanks

Reply:
Hello @luigimassa which php version are you using ?

I guess that the composer.lock file was updated with a version > php 7.2 
( which is the minimal version we support  as you can see here : https://github.com/friends-of-presta/fop_console )
As the error in the action suggest.
Could you please try the following :
 - revert the composer.json and composer.lock file to their initial states.
 - require your librairies using php7.2 version  ?
 
 Regards

Reply:
Hello, 
Hi revert composer.json and lock file but I added new repository 
-  nette/php-generator
-  vincentlanglet/twig-cs-fixer

Reply:
Hi 
have you read about php7.2 ?
The automatic test fails as with composer dependencies :
![image](https://user-images.githubusercontent.com/7163132/157311607-2f2c7191-cb0a-423c-8e64-16e2a8a3b7fe.png)